lab1/lab1.rst

```diff
@@ -0,0 +1,396 @@
+
+.. _outputs-1:
+
+Outputs
+~~~~~~~
+
+-  *ROMCombinationalOutput*: The concatenation of *Out2-0*, with *Out2*
+   as the top-most (most significant) bit and *Out0* as the bottom-most
+   (least significant) bit.
+
+Testing
+=======
+
+You will be provided with the following circuits to facilitate testing.
+
+-  **Combinational Input**: Generates the inputs for the combinational
+   circuit.
+
+   -  **Inputs**:
+
+      -  *Clock*: The system clock.
+
+   -  **Outputs**:
+
+      -  *Input*: The concatenation of the input signals *In7-In0* to
+         the combinational circuit.
+
+
+You will also be provided with the following log files to test if your
+circuits are correct:
+
+-  **part1correct.txt**
+
+   -  The log file containing the correct outputs for the combinational
+      logic circuit using combinational logic.
+
+   -  The X’s in the file indicate don’t cares.
+
+-  **part2correct.txt**
+
+   -  The log file containing the correct outputs for the combinational
+      logic circuit using a ROM.
+
+   -  The X’s in the file indicate don’t cares.
+
+We will be testing your code using Logisim’s logging feature. To log the
+results of your program, do the following:
+
+#. Attach a probe or pin to the wires that you want to log, and give it
+   a name.
+
+#. Click Simulate → Logging.
+
+#. In the Selection tab, select the signals you want to log.
+
+#. Click on the File tab.
+
+#. Select a file to log the signals to.
+
+You will need to create three separate log files, one for each
+sub-circuit:
+
++-----------------------+-----------------------+-----------------------+
+| Signal Name           | Radix                 | Description           |
++-----------------------+-----------------------+-----------------------+
+| Input                 | 2                     | The concatenation of  |
+|                       |                       | In7-0.                |
++-----------------------+-----------------------+-----------------------+
+| CombinatonalOutput    | 2                     | The concatenation of  |
+|                       |                       | Out2-0 from the       |
+|                       |                       | combinational         |
+|                       |                       | circuit.              |
++-----------------------+-----------------------+-----------------------+
+
++-----------------------+-----------------------+-----------------------+
+| Signal Name           | Radix                 | Description           |
++-----------------------+-----------------------+-----------------------+
+| Input                 | 2                     | The concatenation of  |
+|                       |                       | In7-0.                |
++-----------------------+-----------------------+-----------------------+
+| ROMCombinationalOutpu | 2                     | The concatenation of  |
+| t                     |                       | Out2-0 from the ROM   |
```

Comment:
T should be on the other line... that's annoying :/. Sorry.

Reply:
No yea I noticed that later and it looks silly on the table, my b.